docs/source/user-guide/nlp/quickstarts/plot_getting_started.py

```diff
@@ -0,0 +1,213 @@
+"""
+
+#%%
+# Load Data & Create TextData Objects
+# ===================================
+# For the purpose of this guide we'll use a small subset of the
+# `tweet emotion <https://github.com/cardiffnlp/tweeteval>`__ dataset:
+
+# Imports
+from deepchecks.nlp import TextData
+from deepchecks.nlp.datasets.classification import tweet_emotion
+
+# Load Data
+train, test = tweet_emotion.load_data(data_format='DataFrame')
+train.head()
+
+#%%
+#
+# We can see that we have the tweet text itself, the label (the emotion) and then some additional metadata columns.
+#
+# We can now create a :class:`TextData <deepchecks.nlp.TextData>` object for each of the train and test dataframes.
+# This object is used to pass your data to the deepchecks checks.
+#
+# To create a TextData object, the minimum required arguments is the text itself, but passing only the text
+# will prevent multiple checks from running. In this example we'll pass the label as well and also provide
+# Metadata (the other columns in the dataframe) which we'll use later on in the guide. Finally, we'll also
+# explicitly set the index, as in our case it's not the default index.
+#
+# .. note::
+#    The label column is optional, but if provided you must also pass the task_type argument, so that deepchecks
+#    will know how to interpret the label column.
+#
+
+train = TextData(train.text, label=train['label'], task_type='text_classification',
+                 index=train.index, metadata=train.drop(columns=['label', 'text']))
+test = TextData(test.text, label=test['label'], task_type='text_classification',
+                index=test.index, metadata=test.drop(columns=['label', 'text']))
+
+#%%
+# Building a Model
+# ================
+#
+# In this example we'll train a very simple model using the CatBoostClassifier, trained over the embeddings of the
+# tweets. These embeddings where created using the OpenAI GPT-3 model. You can calculate your own embeddings using
+# our :func:`calculate_embeddings_for_text <deepchecks.nlp.utils.calculate_embeddings_for_text>`
+# function. Note that in order to run it you need either an OpenAI API key or have HuggingFace's transformers installed.
+
+from sklearn.metrics import roc_auc_score
+from sklearn.model_selection import train_test_split
+from catboost import CatBoostClassifier
+
+# Load Embeddings and Split to Train and Test
+embeddings = tweet_emotion.load_embeddings()
+train_embeddings, test_embeddings = embeddings[train.index, :], embeddings[test.index, :]
+
+model = CatBoostClassifier(max_depth=2, n_estimators=50, random_state=42)
+model.fit(embeddings[train.index, :], train.label, verbose=0)
+print(roc_auc_score(test.label,
+                    model.predict_proba(embeddings[test.index, :]),
+                    multi_class="ovr", average="macro"))
+
+#%%
+# Running Deepchecks
+# ==================
+#
+# Now that we have our data and model, we can run our first checks!
+#
+# Performance Checks
+# ------------------
+#
+# We'll start by running the
```

Comment:
Consider by starting with the label drift

Reply:
why not prediction drift? More interesting